doc/gov/chapters/chapter01.md

```diff
@@ -11,18 +11,18 @@
+The founding members of the Cloud iNfrastructure Telco Task Force (CNTT) were AT&T, Vodafone, Verizon, Deutsche Telecom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force. CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation Networking in Summer 2019. It now has over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member commmunity and those numbers continue to grow. Together th CNTT community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support Network Function Virtualization (NFV) applications for the Telecom industry as a whole. The working group is collaborating with OPNFV and other open source communities to operationalize and support reference implementations and reference certification platforms. 
 
-In order to establish governing guidelines and procedures by which the CNTT technical community may operate, the development of this document was undertaken by a CNTT working group and developed under the guidance of the CNTT Governance Steering Committee. 
+In order to establish governing guidelines and procedures by which the CNTT technical community may operate, the development of this document was undertaken by a CNTT workstream group and developed under the guidance of the CNTT Governance Steering Committee. 
 
 <a name="1.2"></a>
 ## 1.2 Document Purpose
 
-The goal of this document is to assist CNTT to continue to scale, garner momentum, and plan for the future of CNTT--as outlined in its 3rd Technical Workshop in Antwerp, Belgium, in September 2019. Therefore, the purpose of this document is to define and detail basic CNTT community operations and important structures. It provides a reference of the organization and related work streams, processes, and procedures around which the CNTT Community will be managed; including operational scope, structure, and responsibilities of all groups and individuals associated with CNTT.  
+The goal of this document is to assist CNTT to continue to scale, garner momentum, and plan for the future of CNTT--as outlined in its 3rd Technical Workshop in Antwerp, Belgium, in September 2019. Therefore, the purpose of this document is to define and detail basic CNTT community operations and important structures. It provides a reference of the organization and related work streams, processes, and procedures around which the CNTT Community is managed; including operational scope, structure, and responsibilities of all groups and individuals associated with CNTT.  
```

Comment:
@bfcohen Can we simplify, "... outlined in its 3rd Technical Workshop in Antwerp, Belgium, in September 2019." (too many "in"). 
Maybe as: "... outlined at its 3rd Technical Workshop, held in Antwerp, Belgium, September 2019."

Reply:
Sure.